src/utils.ts

```diff
@@ -140,4 +141,18 @@ export function forceForwardSlashes(aUrl: string): string {
     return aUrl
         .replace(/\\\//g, '/') // Replace \/ (unnecessarily escaped forward slash)
         .replace(/\\/g, '/');
+}
+
+export function getUrlFromConfig(folder: vscode.WorkspaceFolder, config: vscode.DebugConfiguration): string {
+    let outUrlString:string = '';
```

Comment:
You don't really need to define the type when you set it directly like this. Typescript will figure it out and it's easy to see. Same in other places.